docs/overview/migration/MigrationV5.stories.mdx

```diff
@@ -47,6 +47,7 @@ Here's what there is to know about this migration:
   - [Time Picker](#time-picker)
   - [Table Renderer - Expand Column](#table-renderer-expand-column)
   - [Drawer](#drawer)
+  - [Query Builder](#query-builder)
```

Comment:
Regarding the migration guide I think we should also add the renamed types and defaults values in the "Renamed components and types" table to maintain consistency 🙏